confmap/provider/httpprovider/provider_test.go

```diff
@@ -0,0 +1,119 @@
+// Copyright The OpenTelemetry Authors
```

Comment:
Add a test like https://github.com/open-telemetry/opentelemetry-collector/blob/main/confmap/provider/fileprovider/provider_test.go#L32

Reply:
Added.

Reply:
Did you forget to push the commit?

Reply:
I think added at line 138 in provider_test.go?